<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyHealthcareOperator</class>
 <widget class="QDialog" name="ModifyHealthcareOperator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>27</y>
     <width>322</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>239</width>
     <height>175</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="firstNameLabel">
      <property name="text">
       <string>Prénom :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="firstNameLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="typeLabel">
      <property name="text">
       <string>Type :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="typeBox">
      <item>
       <property name="text">
        <string>Médecin A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Médecin B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kinésithérapeute</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Informaticien</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Infirmière</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Psychologue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Radiologue</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="loginLabel">
      <property name="text">
       <string>Login :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="loginLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Mot de passe :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="passwordLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="modifyButton">
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>322</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>37</y>
     <width>322</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajout d'un personnel de soin</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
